docs/template-google-cloud.md

```diff
@@ -0,0 +1,22 @@
+In order to deploy to Google Cloud Platform, GCP, you first need to create a google cloud project.
+
+## Creating an account
+
+Creating an account is simple and you get some free credits when you sign up try GCP out. Go to https://cloud.google.com/ to sign up for an account. 
+
+## Creating a project
+
+A `project` on GCP is used to organize resources that belongs together one way or another. Creating a project is as simple as just clicking a button an write down the name of the project. If your `project name` is unique across GCP it will also be your `project id`, but if there are more than one project named the same you need to figure out your `project id`. You find the `project id` in the [Home dashboard](https://console.cloud.google.com/home/dashboard) in the GCP console. 
+
+## Setup gcloud CLI tool
+
+The command line interface for GCP is what most people use to interact with the platform instead of clicking in the UI. To get started with the tool you need to install it following the instructions here: https://cloud.google.com/sdk/install.
+
+When you have installed `gcloud` you need to authenticate and also set the default project to work with:
+
+* Authenticate: `gcloud auth login`
+* Set project: `gcloud config set project PROJECT_ID`
+
+## Enable billing
```

Comment:
The question is whether one does need any paid service to run the SAFE template on GCP? If not, then I'd change this section to encourage more users: if they see they can try it for free we get much better adoption.

If we don't need any paid service for the basic template, change the title of section to `Run for free` and in the content emphasise that you don't need to pay for trying out SAFE template on GCP, and afterwards we can mention that billing needs to be set up only if adding any additional (paid) services

Reply:
I’m a little bit unsure about those aspect since I already have my accounts set up. I can check with a new email if I can register without adding billing, or ask around.

Reply:
I’m a little bit unsure about those aspect since I already have my accounts set up. I can check with a new email if I can register without adding billing, or ask around.

Reply:
@theimowski, did you manage to test it without adding a credit card?